### DISCUSSION TITLE
Crash while building new calibration table for YOLOv4 INT8 precision - failed to read calibration image

### DISCUSSION THREAD

**Discussion by zukelah** [enazoe/yolo-tensorrt#143] [Q&A] [answered]
Hi all,

I'm trying to build yolov4 engine with INT8 precision. 

Output from the test_dll.exe:

File does not exist : ../configs/yolov4-kINT8-batch1.engine
Loading pre-trained weights...
Loading complete!
      layer               inp_size            out_size       weightPtr
(1)   conv-bn-mish      3 x 416 x 416      32 x 416 x 416    992
(2)   conv-bn-mish     32 x 416 x 416      64 x 208 x 208    19680
(3)   conv-bn-mish     64 x 208 x 208      64 x 208 x 208    24032
(4)   route                  -             64 x 208 x 208    24032
(5)   conv-bn-mish     64 x 208 x 208      64 x 208 x 208    28384
(6)   conv-bn-mish     64
.
.
.
(160) conv-bn-leaky   512 x  13 x  13    1024 x  13 x  13    64168030
(161) conv-linear    1024 x  13 x  13     255 x  13 x  13    64429405
(162) yolo            255 x  13 x  13     255 x  13 x  13    64429405
File does not exist : ../configs/yolov4-kINT8-batch1.engine
Building the TensorRT Engine...
New calibration table will be created to build the engine

Then the program crashed as it failed to read the calibration images listed in the configs\calibration_images.txt. 

WARNING: couldn't find: ./data\imgs\000000000009.jpg while loading: ../configs/calibration_images.txt
WARNING: couldn't find: ./data\imgs\000000000025.jpg while loading: ../configs/calibration_images.txt
WARNING: couldn't find: ./data\imgs\000000000030.jpg while loading: ../configs/calibration_images.txt
WARNING: couldn't find: ./data\imgs\000000000034.jpg while loading: ../configs/calibration_images.txt

Where can I get these calibration images? Or can we skip the new calibration table creation?

Thank you in advance.

**Comment by enazoe** [comment, marked as answer]
the calibration images is the subset of your train dataset; can not skip the table creation

**Reply by zukelah** [reply]
Thanks.. it works now :)
